share/picongpu/examples/Voronoi_probalistic/test_voronoi_probalistic.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+VOPONOI_PLUGIN_RUN_STR="TBG_voronoi="--e_randomizedMerger.period 100 --e_randomizedMerger.minParticlesToMerge 100""
```

Comment:
I do not understand why we need this script and why this script is hijacking the config-files from the example KelvinHelmholtzreducted example which is not part of this PR. 

IMO this script should not be checked shipped with this PR.